setup.py

```diff
@@ -31,7 +31,7 @@
     packages=["ariadne"],
     include_package_data=True,
     install_requires=[
-        "graphql-core>=3.0.0,<3.1.0",
+        "graphql-core>=3.0.0",
```

Comment:
```suggestion
        "graphql-core>=3.1.0",
```

Reply:
do we want to force 3.1 on people though?

Reply:
ah, shit. its breaking change. nvm. ya we do